protocol/v2/ssv/runner/metrics/metrics.go

```diff
@@ -229,3 +262,21 @@ func (cm *ConsensusMetrics) RoleSubmissionFailed() {
 		cm.rolesSubmissionFailures.Inc()
 	}
 }
+
+// StartBeaconData stores data start time.
+func (cm *ConsensusMetrics) StartBeaconData() {
+	if cm != nil {
+		cm.beaconDataStart = time.Now()
+	}
+}
+
+// EndBeaconData sends metrics for data duration.
+func (cm *ConsensusMetrics) EndBeaconData() {
+	if cm != nil && cm.beaconData != nil && !cm.beaconDataStart.IsZero() {
+		duration := time.Since(cm.beaconDataStart)
+		cm.beaconDataDuration = duration
+		log.Printf("setting beaconDataDuration to %v (started: %s)", duration, cm.beaconDataStart)
+		cm.beaconData.Observe(duration.Seconds())
+		cm.beaconDataStart = time.Time{}
+	}
```

Comment:
imo its better to do `if cm == nil { return }`

Reply:
> imo its better to do `if cm == nil { return }`

i think this condition should not even exist 😆 

instead of checking for nil, we can just use a `noopConsensusMetrics` implementation 

however, this is how the other methods are implemented so we can leave this for metrics refactor